src/genpy/rostime.py

```diff
@@ -180,7 +180,9 @@ def __ne__(self, other):
     def __cmp__(self, other):
```

Comment:
Arguably the real issue here is implementing `__cmp__` in python 3 at all. If `cmp` doesn't exist, then `__cmp__` will never be called. So I don't understand what this patch is fixing.